src/GovUk.Education.ExploreEducationStatistics.Admin/Migrations/ContentMigrations/20201112030631_EES1319UpdateDataBlockChartLegend.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace GovUk.Education.ExploreEducationStatistics.Admin.Migrations.ContentMigrations
+{
+    public partial class EES1319UpdateDataBlockChartLegend : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
```

Comment:
I got an error locally when applying this migration:
```
info: Microsoft.EntityFrameworkCore.Migrations[20402]
      Applying migration '20201112030631_EES1319UpdateDataBlockChartLegend'.
fail: Microsoft.EntityFrameworkCore.Query[10100]
      An exception occurred while iterating over the results of a query for context type 'GovUk.Education.ExploreEducationStatistics.Content.Model.Database.ContentDbContext'.
      Newtonsoft.Json.JsonSerializationException: Error converting value "(null)" to type 'System.Nullable`1[GovUk.Education.ExploreEducationStatistics.Common.Model.Chart.ChartLegendPosition]'. Path 'Legend.Position', line 1, position 1979.
       ---> System.ArgumentException: Requested value '(null)' was not found.
```

Reply:
Good catch, should be fixed now!